### DISCUSSION TITLE
Batch learning/predicting for the Hoeffding Adaptive Tree Classifier

### DISCUSSION THREAD

**Discussion by gintriago12** [online-ml/river#709] [Q&A] [answered]
Hi,

The Hoeffding Adaptive Tree classifier have the methods learn_many() and predict_many(), is there any way to enable it? Any guidance/help is welcomed.

Best,

**Comment by smastelini** [comment, marked as answer]
Hi @gintriago12. Unfortunately, at the time, our tree models only support an instance-by-instance learning regimen. You will need to loop over the instance batches.

Cheers,

**Reply by gintriago12** [reply]
Thanks @smastelini
